### DISCUSSION TITLE
Is it possible to tile a square png image over the tasklist?

### DISCUSSION THREAD

**Discussion by ghost** [awesomeWM/awesome#3926] [Q&A]
I am currently using a wide `1920x28px` image as background to the `awful.widget.tasklist`. This wide image can be reproduced by a repeated pattern of `100x28px`.

The problem with the wide image is that it works fine on screens with resolution 1920, but not on higher or lower resolutions.

I think this problem can be solved if tiling an image was allowed (for example, tile 90% of the tasklist). Is this possible to do please?

**Comment by actionless** [comment]
yes, i remember doing that in pre-hidpi epoch, but i don't have the code anymore
